src/DssSpell.sol

```diff
@@ -86,98 +71,30 @@ contract DssSpellAction is DssAction {
-            )
-        );
-
-        // ---------------------- SPF Funding Transfers ------------------------
-        // https://forum.makerdao.com/t/mip55c3-sp6-legal-domain-work-on-greenlit-collateral-bibta-special-purpose-fund/17166
-        // https://vote.makerdao.com/polling/QmdaG8mo#vote-breakdown
-        DssExecLib.sendPaymentFromSurplusBuffer(BIBTA_WALLET, 50_000);
-        DssExecLib.sendPaymentFromSurplusBuffer(MIP65_WALLET, 30_000);
-
-        // ------------------- GRO-001 MKR Stream Clean-up ---------------------
-        // https://forum.makerdao.com/t/executive-inclusion-gro-001-mkr-vesting-stream-clean-up/17820
-        VestLike mkrVest = VestLike(
-            DssExecLib.getChangelogAddress("MCD_VEST_MKR_TREASURY")
-        );
-        mkrVest.vest(2);
-
-        // -------------------- Update HVB Legal Documents ---------------------
-        // https://forum.makerdao.com/t/poll-inclusion-request-hvbank-legal-update/17547
-        // https://vote.makerdao.com/polling/QmX81EhP#vote-breakdown
-        bytes32 ilk                      = "RWA009-A";
-        address MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress(
-            "MIP21_LIQUIDATION_ORACLE"
-        );
-
-        ( , address pip, uint48 tau, ) = RwaLiquidationLike(
-            MIP21_LIQUIDATION_ORACLE
-        ).ilks(ilk);
-
-        require(pip != address(0), "Abort spell execution: pip must be set");
-
-        // Init the RwaLiquidationOracle to reset the doc
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
-            ilk,       // ilk to update
-            0,         // price ignored if init() has already been called
-            DOC,       // new legal document
-            tau        // old tau value
-        );
+        // --- MKR Transfers ---
+        GemLike mkr = GemLike(DssExecLib.mkr());
+
+        // RWF-001 - 20 MKR - 0x96d7b01Cc25B141520C717fa369844d34FF116ec
+        // https://mips.makerdao.com/mips/details/MIP40c3SP38
+        mkr.transfer(RWF_WALLET, 20.00 ether);
+
+        // CES-001 - 966.49 MKR - 0xD740882B8616B50d0B317fDFf17Ec3f4f853F44f
+        // https://mips.makerdao.com/mips/details/MIP40c3SP30
+        mkr.transfer(CES_OP_WALLET, 966.49 ether);
+
+        // RISK-001 175 MKR - 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c
+        // https://mips.makerdao.com/mips/details/MIP40c3SP25
+        mkr.transfer(RISK_WALLET_VEST, 175.00 ether);
+        // ---------------------
+
+        // Increase Starknet Bridge Deposit Limit from 50 DAI to 1000 DAI
+        // https://vote.makerdao.com/polling/QmbWkTvW
+        StarknetLike(DssExecLib.getChangelogAddress("STARKNET_DAI_BRIDGE")).setMaxDeposit(1000 * WAD);
+
+        // --- Chainlog Update ---
+        // https://forum.makerdao.com/t/28th-september-executive-updating-the-proxy-actions-end-cropper-address/18057
+        DssExecLib.setChangelogAddress("PROXY_ACTIONS_END_CROPPER", PROXY_ACTIONS_END_CROPPER);
```

Comment:
I just want to double check as I'm not aware, but is this filed() anywhere other than in the chainlog?  That is, is anything pointed at this other than the chainlog?

Reply:
Just the chainlog as the regular proxy actions.